concourse/pipeline.yml

```diff
@@ -105,10 +105,12 @@ jobs:
             args:
               - -c
               - |
-                set -ue
-                cat .git/committer > ../committer-details/last-committer
-                postgres -p 5432 -D /usr/local/var/postgres >&1 &
-                bundle exec rake
+                set -eu
+                service postgresql start
+                su - postgres -c "psql -c \"create role root with createdb login password 'password';\""
+                DATABASE_URL="postgres://root:password@localhost:5432/coronavirus_form_development" bundle exec rails db:setup
+                DATABASE_URL="postgres://root:password@localhost:5432/coronavirus_form_development" bundle exec rails db:migrate
+                DATABASE_URL="postgres://root:password@localhost:5432/coronavirus_form_development" bundle exec rake
```

Comment:
Looks good! Only comment/question was if it would be possible to do `export DATABASE_URL="postgres://root:password@localhost:5432/coronavirus_form_development"` so as not to repeat it? or does it need to be defined for the rails commands?

Reply:
yeah much better idea, will give it a go!